synfig-studio/src/gui/resources/ui/vectorizer_settings.glade

```diff
@@ -189,6 +186,9 @@
                 <property name="top-attach">3</property>
               </packing>
             </child>
+            <style>
+              <class name="color-grid"/>
```

Comment:
`dialog-main-content` would suit better LOL

`color-grid` was created for Color chooser dialog